### PR TITLE
Fixes for cell switcher

### PR DESCRIPTION
**Pull request by stefnotch**
- Fixes a bug with the up and down arrow keys in monaco not working when there is more than one line of coode
- Jumps to the first/last line when switching between cells
- Fixes the next cell getting focused when adding a new one using alt+enter

This code should work, but I'm always super happy about feedback of any kind :) I'm also pretty sure that not everything is as neat as it could be.

### CODE REVIEW

File: src/components/editor/contentEditor.ts [hunk 1]
```diff
@@ -31,18 +31,13 @@ export class StarboardContentEditor extends LitElement {
     return this;
   }
 
-  constructor(content: ContentContainer, runtime: Runtime, opts: { focusAfterInit?: boolean } = {}) {
+  constructor(content: ContentContainer, runtime: Runtime, opts: {} = {}) {
```
Comment (gzuidhof) [inline src/components/editor/contentEditor.ts:34]: Nit: we may as remove the opts if it's empty

File: src/runtime/create.ts [hunk 2]
```diff
@@ -132,21 +132,20 @@ export function setupRuntime(notebook: StarboardNotebookElement): Runtime {
...
+          const next = rt.dom.cells[idxOfCell - 1];
+          if (next) next.focusEditor({ position: "end" });
         });
       } else if (focus === "next") {
         window.setTimeout(() => {
-          const next = cellElements[idxOfCell + 1];
+          const next = rt.dom.cells[idxOfCell + 1];
           if (next) {
-            next.focusEditor();
+            console.log("focusing next cell");
```
Comment (gzuidhof) [inline src/runtime/create.ts:147]: Remove debug print

**Review by gzuidhof — APPROVED** [review]
Thank you :) It looks great, I only left some tiny comments, feel free to merge when you are happy!

File: src/runtime/create.ts (hunk 2 above)
Comment (stefnotch) [reply src/runtime/create.ts:147]: Oh thanks for catching that, totally missed it when committing the files

**Comment by stefnotch** [comment]
Okay, I took care of them! :) I think this is ready for merging (I can't do it by the way).

Implementing the stuff with the focused cell is something I'll possibly do in a future pull request

**Pull request merged**